v2/contribute/decisions/session/0006-token-header-names-follow-cookie-names.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: "0006"
```

Comment:
missed ADRs:
- A version of getSession which takes just the access token and not the req / response objects.
- Allowing users to pass in a `checkCore` boolean (or whatever it was called) during access token verification. This would essentially replace the blacklisting option in the core.